test/models/test_rbm.py

```diff
@@ -56,11 +56,11 @@ def test_RBMSymm(use_hidden_bias, use_visible_bias, symmetries):
     vmc.advance(1)
 
 
-@pytest.mark.parametrize("use_bias", [True, False])
```

Comment:
can't we test both partiy and bias?

Reply:
Of course, I was just worried there was too many tests. Also, the bias gets tested in DenseEquivariant so it felt the most redundant. I can change it back of course.

Reply:
Ha, no, then it's ok